doc/tb3_tutorial/README.md

```diff
@@ -0,0 +1,573 @@
+    source ~/.bashrcros2 launch turtlebot3_gazebo turtlebot3_world.launch.py
+
+
+![enter image description here](https://github.com/mlherd/ros2/blob/master/img/gazebo_turlebot1.PNG?raw=true)
+
+This will start Gazebo and load the world model and robot model which means that we will see a 3D simulated world and simulation of our Turtlebot Waffle.
+
+Now, you should be able control the Turtlebot with your keyboard buttons. Make sure you are the keyboard teleop node terminal when pressing the buttons to control the Turtlebot. Otherwise, Turtlebot won't receive any velocity commands.
+
+![enter image description here](https://github.com/mlherd/ros2/blob/master/img/gazebo_turtlebot2.PNG?raw=true)
+
+*Note: After using source you may experience some delay in the terminal (some times a few minutes). No worries, it is normal.*
+
+# Testing Navigation 2
+
+## Testing Navigation 2 in Gazebo
+
+Now, we know all the packages and ROS tools we need to use the Navigation stack work, so it is time to test Navigation 2. Before we do anything, in order to save time, let's export all the source commands and environment variables to the .bashrc file. 
+
+Open a new terminal and open your .bashrc file which is located in your home directory. You can use any one of your favorite text editors, just remember to have  super user rights to be able to modify the file. I will use gedit to edit the file.
+
+    sudo gedit ~/.bashrc
+
+Add the following lines to the end of the file
+
+    source ~/ros2_all_ws/ros2_ws/install/setup.bash  
+    source ~/ros2_all_ws/turtlebot3/install/setup.bash source
+    source ~/ros2_all_ws/navigation2/install/setup.bash source
+    source ~/ros2_all_ws/navstack_dependencies_ws/install/setup.bash
+    export TURTLEBOT3_MODEL=waffle
+    export GAZEBO_MODEL_PATH=$GAZEBO_MODEL_PATH:~/ros2_all_ws/turtlebot3_ws/src/turtlebot3/turtlebot3_simulations/turtlebot3_gazebo/models
+
+save the file and close it. Whenever we open a new terminal, this script will be executed. Please note that this will will make your terminal's start up time considerably longer, so when you are done with this tutorial you may want to remove these lines from your .bashrc file.
+
+Let's launch Gazebo with the world model first, open a new terminal and type
+
+    gazebo --verbose -s libgazebo_ros_init.so /home/<user_name>/ros2_all_ws/turtlebot3_ws/src/turtlebot3/turtlebot3_simulations/turtlebot3_gazebo/worlds/turtlebot3_worlds/waffle.model
+
+Don't forget to change <user_name> to your actual user name. For example, after typing your user name it should look like this,
+
+    gazebo --verbose -s libgazebo_ros_init.so /home/melih/ros2_all_ws/turtlebot3_ws/src/turtlebot3/turtlebot3_simulations/turtlebot3_gazebo/worlds/turtlebot3_worlds/waffle.model
+
+
+![enter image description here](https://github.com/mlherd/ros2/blob/master/img/gazebo_turlebot1.PNG?raw=true)
```

Comment:
We should move all these images to this repo. They can be put in a subdirectory here.

Reply:
I didn't want to upload all the images to the repo since it can make the size of the repo larger (about 8mb). If it is not a problem, I can move the images to a directory.